pkgs/applications/misc/freeplane/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchurl, unzip, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  version = "1.8.11";
+  pname = "freeplane";
+
+  src = fetchurl {
+    url =
+      "https://sourceforge.net/projects/freeplane/files/freeplane%20stable/freeplane_bin-${version}.zip/download";
```

Comment:
```suggestion
    url = "https://sourceforge.net/projects/freeplane/files/freeplane%20stable/freeplane_bin-${version}.zip/download";
```

Reply:
This change is forced by `nixfmt`

Reply:
Please don't follow that suggestion here.

Reply:
I didn't, this is already outdated.